tests/attacks/blocked-flop.mcd

```diff
@@ -0,0 +1,36 @@
+STEPS ( DEPLOY-PRELUDE )
+STEPS ( ATTACK-PRELUDE )
+
+// Attack Sequence
+// ---------------
+
+// Set sump (determines DAI bid size for flop auctions)
+transact ADMIN Vow . file sump 50
+
+// Set dump (determines initial MKR lot size for flop auctions)
+transact ADMIN Vow . file dump 30
+
+// Set tau (flop auction lifetime in seconds)
+transact ADMIN Flop . file tau 3600
+
+// Use suck to give Bobby some DAI, which conveniently also creates enough free, unbacked debt for 2 flop auctions
+transact ADMIN Vat . suck Vow "Bobby" 100
+
+// Start a flop auction
+transact "Bobby" Vow . flop
+
+// Bobby must hope on the Flop contract to be able to bid
+transact "Bobby" Vat . hope Flop
+
```

Comment:
I think this one makes sense in the ATTACK-PRELUDE.

Reply:
Hmmm seems that we've already put a bunch of `hope` calls in there anyway, so might as well stay consisten.